runner/app/pipelines/text_to_image.py

```diff
@@ -94,6 +94,8 @@ def __init__(self, model_id: str):
             self.ldm = AutoPipelineForText2Image.from_pretrained(model_id, **kwargs).to(
                 torch_device
             )
+        if os.environ.get("LOWVRAM"):
```

Comment:
L94 will move the pipeline to the GPU when cuda is available (`torch_device` gets set to cuda) via the `.to()` call. Based on the [diffusers docs](https://huggingface.co/docs/diffusers/main/en/optimization/memory#cpu-offloading) it looks like you should *not* move the pipeline to the GPU first if either CPU or model offloading is used. So, if `LOWVRAM` is enabled you'd probably want to do something like this:

```
self.ldm = AutoPipelineForText2Image.from_pretrained(model_id, **kwargs)

if os.environ.get("LOWVRAM"):
    # Enable CPU or model offloading
else:
    self.ldm.to(torch_device)
```

Reply:
There doesnt seem to be a performance or RAM difference when putting the `.to()` call before it but if the docs say not to then I will make that modification

Reply:
I'm also running into issues with using the SFAST flag with the LOWRAM flag, although it seems the SFAST flag is not working anymore by default. 

Getting:
```
  File "/root/.pyenv/versions/3.11.8/lib/python3.11/site-packages/sfast/jit/overrides.py", line 21, in __torch_function__
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
NotImplementedError: Cannot copy out of meta tensor; no data!
```

Either way I'm going to do some more testing to make sure the LOWRAM flag is put in the correct place and does not effect the SFAST flag. I'll close this for now and do everything in a single commit.

Reply:
Ah I wonder if stable-fast only works if the entire pipeline is loaded to the CUDA device.